Greenstand-Token.md

```diff
@@ -0,0 +1,592 @@
+
+
+## Fungible Tokens
+
+A Greenstand Token may easily convert into any number of fungible tokens, or tokens of the same value, using any area of customized Relative Value Indexes.
+
+Fungible tokens enable global (network-wide) standards for exchanging NFTs and facilitate using tokens in local trade.
+
+
+## Relative Value Index(s) (RVI)
+
+Each Greenstand Token is unique and different by nature. RVI’s are central to the conversion of these NFTs into fungible tokens.
+
+Each token’s unique attributes may be measured using a Relative Value Index to place ‘weight’ or a ‘score’ of value relative to any other NFT. RVI’s account for factors such as social impact, biodiversity, carbon sequestration, relative cooling effects, and other co-benefits.
+
+Customized RVIs, may be open-source or proprietary and may mint wide varieties of fungible tokens. Certain RVI’s will be readily accepted or accredited, while others may be limited or convoluted in scope.
+
+The Greenstand Network holds a master RVI and a corresponding fungible token as a network currency defined by the consensus of Greenstand Nodes.
+
+
+
+
+### Key concepts
+
+
+
+* RVIs allow comparative measurements for valuing any NFT against other NFTs based on attributes attached to each token. (For example, when using an RVI that only accounts for carbon ton/time, the value of a token derived from a capture containing 60 days of a tree’s growth may be proportionally more valuable than a capture containing 30 days of a tree’s growth.)
+* RVIs can be created and applied to any NFT in the system based on an array of publicly available attributes attached to that NFT.
+* RVIs create subsets of fungible tokens.
+* The sealing nodes in the network use a single agreed-upon Master RVI to create the network’s native currency.
+
+
+### RVI USES
+
+
+
+* Purchasing tokens from impact creators (Planter/grower)
+* Communicating value and developing incentives for specific positive behaviors, such as fostering diversity versus monocropping or growing high-value indigenous plants versus invasive plants.
+* Accounting
+* Creates a primary fungible unit, TRing, to operate and transact on the network.
+* Valuing impact sourced more directly from responsible entities. (Note: Tokens derived from entities that claim all results from an area are distinguishable from individual growers as landowners or other impact creators.)
+
+
+## Token States
+
+A Greenstand Token’s life cycle contains five distinct states that affect the token’s ability to be traded and have its attributes changed. The states are Basic, Mature, Blocked, Immutable, and Locked.
```

Comment:
A figure here would be helpful, showing the token states, and what is needed to move among the states.